tests/e2e/utils/cognito_bootstrap/cognito_resources_cleanup.py

```diff
@@ -62,6 +64,13 @@ def clean_root_domain(domain_name, hosted_zone_id, subdomain_hosted_zone):
     except Exception:
         pass
 
+def delete_policy(arn: str, region: str):
+    try:
+        policy = IAMPolicy(arn=arn, region=region)
+        policy.delete()
+    except Exception:
+        pass
```

Comment:
Is there some information we can provide here besides just passing?

Reply:
This code is only suppressing the error. Exception message is printed in the `delete()` method